src/AppBundle/Controller/RfXAnalysisController.php

```diff
@@ -0,0 +1,214 @@
+<?php
+/**
+ * This file contains the code that powers the RfX Analysis page of xTools.
+ *
+ * @category RfXAnalysis
+ * @package  AppBundle\Controller
+ * @author   XTools Team <xtools@lists.wikimedia.org>
+ * @license  GPL 3.0
+ * @link     http://xtools.wmflabs.org/rfa
+ */
+
+namespace AppBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Xtools\ProjectRepository;
+use Xtools\RFA;
+
+/**
+ * Class RfXAnalysisController
+ *
+ * @category RfXAnalysis
+ * @package  AppBundle\Controller
+ * @author   XTools Team <xtools@lists.wikimedia.org>
+ * @license  GPL 3.0
+ * @link     http://xtools.wmflabs.org/rfa
```

Comment:
I put it in because it's required in order to pass Travis.  Maybe @link should be the docs?  I maintain the mailing list, it's dead without even spam anymore.